src/blocks/body.css

```diff
@@ -0,0 +1,6 @@
+body {
```

Comment:
```suggestion
.body {
```
Краще стилізувати body по класу а не по тегу

Reply:
fixed
UPD